lib/use_case/question/create.rb

```diff
@@ -11,6 +11,7 @@ class Create < UseCase::Base
       option :content, type: Types::Coercible::String
       option :anonymous, type: Types::Params::Bool, default: proc { false }
       option :author_identifier, type: Types::Coercible::String | Types::Nil
+      option :direct, type: Types::Params::Bool, default: proc { true }
```

Comment:
on second thought... I think this option is not needed here, as 1-1 questions (which this usecase is for) can only be direct anyways.

Reply:
I added the option just in case, just so we have the option to still do non-direct ones in the future, for whatever reason.

Might not be specifically needed right now, but I found this better than removing entire methods tied to other logic here.

Reply:
fair enough!